File: csharp/raindrops/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        string speak = "";

        if (number % 3 == 0) {
            speak += "Pling";
        }
        if (number % 5 == 0) {
            speak += "Plang";
        }
        if (number % 7 == 0) {
            speak += "Plong";
        }
        if (speak == "") {
            return number.ToString();
        }
        return speak;
    }
}

File: typescript/raindrops/raindrops.ts
export default class Raindrops {
  convert(input: number): string {
    let response: string = ""

    if (input % 3 === 0) { response += "Pling" }
    if (input % 5 === 0) { response += "Plang" }
    if (input % 7 === 0) { response += "Plong" }
    if (response.length === 0) { response = input.toString() }

    return response
  }
}